### ISSUE TITLE
jupyverse singleuser server doesn't appear to be working

### ISSUE THREAD

**Issue by minrk** [jupyterhub/jupyterhub#4554]
### Bug description

Tests with JUPYTERHUB_SINGLEUSER_APP=jupyverse have started to fail, due to https://github.com/jupyter-server/jupyverse/issues/348

cc @davidbrochart 

#### Expected behaviour

single-user server can start with jupyverse

#### Actual behaviour

single-user server with jupyverse exits immediately without starting a server or providing any feedback

### How to reproduce

```
pip install jupyverse[jupyterlab,auth-jupyterhub]
pip install -e .

JUPYTERHUB_SERVICE_URL=http://127.0.0.1:8998 JUPYTERHUB_SINGLEUSER_APP=jupyverse python3 -m jupyterhub.singleuser
```

which will exit immediately instead of starting a server.

### Your personal set up

<details>
```
aiosqlite==0.19.0
alembic==1.12.0
annotated-types==0.5.0
anyio==3.7.1
appnope==0.1.3
argon2-cffi==23.1.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
asgiref==3.7.2
asphalt==4.12.0
asphalt-sqlalchemy==5.0.1
asphalt-web==1.2.1
asttokens==2.4.0
async-generator==1.10
async-lru==2.0.4
async-timeout==4.0.3
asyncio-extras==1.3.2
attrs==23.1.0
Babel==2.12.1
backcall==0.2.0
beautifulsoup4==4.12.2
bleach==6.0.0
build==1.0.0
certifi==2023.7.22
certipy==0.1.3
cffi==1.15.1
charset-normalizer==3.2.0
click==8.1.7
comm==0.1.4
cryptography==41.0.3
debugpy==1.6.7.post1
decorator==5.1.1
defusedxml==0.7.1
executing==1.2.0
fastapi==0.103.1
fastjsonschema==2.18.0
fps_auth_jupyterhub==0.2.4
fps_contents==0.2.4
fps_frontend==0.2.4
fps_jupyterlab==0.2.4
fps_kernels==0.2.4
fps_lab==0.2.4
fps_nbconvert==0.2.4
fps_terminals==0.2.4
fps_yjs==0.2.4
fqdn==1.5.1
greenlet==2.0.2
h11==0.14.0
httpcore==0.17.3
httpx==0.24.1
idna==3.4
ipykernel==6.25.2
ipython==8.15.0
isoduration==20.11.0
jedi==0.19.0
Jinja2==3.1.2
json5==0.9.14
jsonpointer==2.4
jsonschema==4.19.0
jsonschema-specifications==2023.7.1
jupyter-events==0.7.0
jupyter-lsp==2.2.0
jupyter-telemetry==0.1.0
jupyter-ydoc==1.0.2
jupyter_client==8.3.1
jupyter_core==5.3.1
jupyter_server==2.7.3
jupyter_server_terminals==0.4.4
-e git+ssh://git@github.com/jupyterhub/jupyterhub.git@f4163069139251c2be2d7752607518e97bd31079#egg=jupyterhub
jupyterlab==4.0.5
jupyterlab-pygments==0.2.2
jupyterlab_server==2.24.0
jupyverse==0.2.4
jupyverse_api==0.2.4
Mako==1.2.4
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib-inline==0.1.6
mdurl==0.1.2
mistune==3.0.1
nbclient==0.8.0
nbconvert==7.8.0
nbformat==5.9.2
nest-asyncio==1.5.7
notebook_shim==0.2.3
oauthlib==3.2.2
overrides==7.4.0
packaging==23.1
pamela==1.1.0
pandocfilters==1.5.0
parso==0.8.3
pexpect==4.8.0
pickleshare==0.7.5
pip-tools==7.3.0
platformdirs==3.10.0
prometheus-client==0.17.1
prompt-toolkit==3.0.39
psutil==5.9.5
ptyprocess==0.7.0
pure-eval==0.2.2
pycparser==2.21
pydantic==2.3.0
pydantic_core==2.6.3
Pygments==2.16.1
pyOpenSSL==23.2.0
pyproject_hooks==1.0.0
python-dateutil==2.8.2
python-json-logger==2.0.7
PyYAML==6.0.1
pyzmq==25.1.1
referencing==0.30.2
requests==2.31.0
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rich==13.5.2
rich-click==1.6.1
rpds-py==0.10.2
ruamel.yaml==0.17.32
ruamel.yaml.clib==0.2.7
Send2Trash==1.8.2
six==1.16.0
sniffio==1.3.0
soupsieve==2.5
SQLAlchemy==2.0.20
stack-data==0.6.2
starlette==0.27.0
terminado==0.17.1
tinycss2==1.2.1
tornado==6.3.3
traitlets==5.9.0
types-python-dateutil==2.8.19.14
typing_extensions==4.7.1
uri-template==1.3.0
urllib3==2.0.4
uvicorn==0.23.2
watchfiles==0.20.0
wcwidth==0.2.6
webcolors==1.13
webencodings==0.5.1
websocket-client==1.6.2
websockets==11.0.3
y-py==0.6.0
ypy-websocket==0.12.2
```

</details>

**Comment by davidbrochart** [comment]
I released jupyverse v0.2.5 which should fix this issue ([PR](https://github.com/jupyter-server/jupyverse/pull/349)).

**Comment by minrk** [comment]
Yup, [fixed](https://github.com/jupyterhub/jupyterhub/actions/runs/6094989365/job/16551112135). Thanks!
